### ISSUE TITLE
Question about editing loss

### ISSUE THREAD

**Issue by ylylyl-sjtu** [VectorSpaceLab/OmniGen#200]
Thanks for such a wonderful work!

In your training code I noticed the loss function for editing task. However, I don't quite get the meaning of the "need_edit_imgs", I suspect that "input_x = input_pixel_values[i]" is enough? What does need_edit_imgs[i] stand for?  Is it a boolean or int?

Thank you!

**Comment by staoxiao** [comment]
`need_edit_imgs[i]` is an index for images for edit task.  Since we train OmniGen on multi-task data, part of the images in one batch data is not for edit task.
